pimpmycause/pimpmycause/templates/index.html

```diff
@@ -218,14 +218,14 @@ <h1 class="pnl__title">Have You Seen</h1>
                                 <img class="summary-img" src=" {% static "img/summary1-how-it-works.png" %}" alt="How it works">
                                 <div class="summary-title">How it works</div>
                                  <div class="summary-info">
-                                    Paul created Pimp My Cause and has led the platform to creating over £15 million of social value, making pro bono marketing accessible.. <a href="#" class="read-more">Read more</a> 
+                                    Paul created Pimp My Cause and has led the platform to creating over £15 million of social value, making pro bono marketing accessible.. <a href="how-it-works/" class="read-more">Read more</a> 
```

Comment:
Should it be like href="/how-it-works/"? Like the others? Or it wouldnt make a difference?